examples/integrations/neo4j/neo4j-se/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Please remove the `native-image` directory unless used

Reply:
actually used and working (on my machine with 20.2)

Reply:
The `reflect-config.json` is empty - e.g. not used. The directory `native-image` only contains `reflect-config.json`, so it can be removed as well if you remove the file

Reply:
Fixed.